actions/molecule-test/README.md

```diff
@@ -0,0 +1,32 @@
+# Molecule Test
+
+This action can be used in the following manner:
+
+```yaml
+jobs:
+  molecule:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Run `molecule test`
+        uses: UCL-MIRSG/.github/actions/molecule-test@v.0.1.0
```

Comment:
Should the ref be `v0.1.0`? And should we keep the version stated in `README.md` up-to-date with the repo tag (currently `v0.21.0`)?

Reply:
I think it's not really possible to keep the README up to date with the repo tag, as a new release is automatically made every time a pr is merged (and I guess that would mean we would need to update the README on every pr). Unless there's a way to automatically keep it up to date?

Reply:
Sorry - I didn't see the workflow for auto-tagging. In that case maybe the README should just say "vx.x.x" to indicte that a version should go there? Sorry for pendantry - I'll approve and you can choose to ignore if you like.

Reply:
yeah that's probably a neater way of doing it, I've updated it now. Thanks for approving!

Reply:
We should probably have it consistent throughout the `README.md`s. The problem is that there isn't a convenient `pre-commit autoupdate` equivalent for actions.

Reply:
yeah consistency would be good, are you happy with the `vx.y.z` convention used in this action?

Reply:
Yeah, just raising a PR as we speak